notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,26 @@
 {
```

Comment:
<p>You can show the final equations as a matrix-vector form:</p><p> </p><p>$\mathbf{A F} = \mathbf{F}_applied$</p><p> </p><p>$\left[\begin{array}</p><p>1/3 &amp; 1/3 &amp; 1 &amp; 0 &amp; 0\\</p><p>1 &amp; -1 &amp; 0 &amp; 1 &amp; 0\\</p><p>...</p><p>\end{array}\right]</p><p>\left[\begin{array}</p><p>T_{BD}\\</p><p>T_{BE}\\</p><p>R_x\\</p><p>R_y\\</p><p>R_z \end{array}\right]$</p><p>=</p><p>\left[\begin{array}</p><p>195\\</p><p>390\\</p><p>...\end{array}\right]$</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>